blog-home.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: blog-home
+template-title: Blog Home
+template-description: Card-based landing page for a blog with media cards, tags, and secondary navigation
+title: Blog
+description: Latest updates on how agencies are serving the American people
+source-domain: usa.gov
+source-url: tbd
+blog-posts:
+    - title: "How USAGov Cracked a Cybercrime Using Free Web Analytics Tools"
+      author: "Sondra Ainsworth and Constance Lu"
+      date: "September 30, 2020"
+      image: https://raw.githubusercontent.com/Bixal/rrt-content/main/assets/img/blog-4.jpg
+      labels:
```

Comment:
Use tags from source content as well. They appear at the bottom of the individual posts: 
![image](https://user-images.githubusercontent.com/5177337/122807523-b09f6f00-d299-11eb-8907-1c8526e72e60.png)